core/src/main/java/org/infinispan/metrics/impl/MetricsCollector.java

```diff
@@ -109,30 +109,40 @@ private static String generateRandomName() {
    }
 
    public Set<Object> registerMetrics(Object instance, Collection<MBeanMetadata.AttributeMetadata> attributes, String namePrefix, String cacheName) {
-      Set<Object> metricIds = new HashSet<>();
+      return doRegisterMetrics(instance, attributes, namePrefix, cacheName, nodeTag);
```

Comment:
```suggestion
      return registerMetrics(instance, attributes, namePrefix, cacheName, null);
```

Reply:
The point was to use this change and the next together, note how I also had suggested to change it to registerMetrics as well. But I misunderstood the second one I guess.

Reply:
Updated the method name.